src/move/tasks/identify_associations.py

```diff
@@ -175,15 +163,9 @@ def prepare_for_continuous_perturbation(
     )
     dataloaders.append(baseline_dataloader)
 
-    baseline_dataset = cast(MOVEDataset, baseline_dataloader.dataset)
+    logger.debug(f"Dataloaders length is {len(dataloaders)}")
 
-    assert baseline_dataset.con_all is not None
-    orig_con = baseline_dataset.con_all
-    nan_mask = (orig_con == 0).numpy()  # NaN values encoded as 0s
-    logger.debug(f"# NaN values: {np.sum(nan_mask)}/{orig_con.numel()}")
-    feature_mask = nan_mask
-
-    return (dataloaders, nan_mask, feature_mask)
+    return dataloaders
```

Comment:
Return changed, but not the type hint of this function

Reply:
Same for `prepare_for_categorical_perturbation`

Reply:
true... I'll fix that